packages/core/test/input.test.ts

```diff
@@ -30,6 +30,28 @@ describe('input', () => {
     expect(spy).toHaveBeenCalledWith(42);
   });
 
+  it('should provide input to the machine from withInput', () => {
+    const spy = jest.fn();
+
+    const machine = createMachine({
+      types: {} as {
+        context: { count: number };
+        input: { startCount: number };
+      },
+      context: ({ input }) => ({
+        count: input.startCount
+      }),
+      entry: ({ context }) => {
+        spy(context.count);
+      }
+    });
+
```

Comment:
Can you add a quick type check here (just for anyone who reads this test) that trying to create an actor from a non-inputted machine will cause a type error?

```suggestion
    // @ts-expect-error
    createActor(machine);
```